Linux packaged version fails to start

I ran `npm run package` and then extracted `dist/tofino-0.1.0-linux-x64.zip` into a new directory. From there, running `./tofino` fails with:

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'immutable'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/fabrice/sandbox/tofino/resources/app/lib/main/reducers/bookmarks-reducers.js:12:18)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

--- a/build/task-package.js
+++ b/build/task-package.js
@@ -16,18 +16,14 @@ const PLATFORM = os.platform();
 
 const IGNORE = [
   // Ignore build stuff
-  '/appveyor.yml',
-  '/branding($|/)',
-  '/build($|/)',
-  '/dist($|/)',
+  '/\/appveyor.yml',
+  '/\/branding($|/)',
+  '/\/build($|/)',
+  '/\/dist($|/)',
   '/electron($|/)',
-  '/gulpfile.js',
   '/README.md',
   '/scripts($|/)',
   '/test($|/)',
-
-  // Ignore source code
-  '/ui($|/)',
 
   // Ignore `/app` and `/shared` once we compile main process code
   // ahead of time, rather than during runtime

